.devhubignore

```diff
@@ -0,0 +1,24 @@
+/.babelrc
+/.devhubignore
+/.dockerignore
+/.editorconfig
+/.env
+/.eslintignore
+/.eslintrc
+.git/
+/.gitignore
+/.prettierrc
+.s2i/
+.vscode/
+/Caddyfile
+/Dockerfile
+/Jenkinsfile
+/LICENSE
+README.md
```

Comment:
you'll need to set this too `/README.md` otherwise it won't be ignored.

Reply:
Fixed.